components/transactions/TransactionDetails.js

```diff
@@ -166,7 +166,7 @@ const TransactionDetails = ({ displayActions, transaction, onMutationSuccess })
               taxInfo: transaction.taxInfo,
               intl,
             })}
-            {transaction.kind !== 'HOST_FEE' && hostFeeTransaction && (
+            {['CONTRIBUTION', 'ADDED_FUNDS'].includes(transaction.kind) && hostFeeTransaction && (
```

Comment:
Doesn't make sense to display this out of the main transaction kind, looks quite weird on related transactions (Tip, Debt, etc ...)